sdks/python/apache_beam/transforms/trigger_test.py

```diff
@@ -850,6 +873,16 @@ def diff(actual, expected):
        | beam.Flatten()
        | beam.ParDo(Check(self.allow_out_of_order)))
 
+    with TestPipeline() as p:
+      # TODO(BEAM-8601): Pass this during pipeline construction.
+      p.options.view_as(StandardOptions).streaming = True
+
+      # We can have at most one test stream per pipeline, so we share it.
+      inputs_and_expected = p | read_test_stream
+      inputs_and_expected | CheckAggregation(beam.GroupByKey())
+      inputs_and_expected | CheckAggregation(beam.CombinePerKey(
```

Comment:
Will this line fail before the combiner lifting issue is fixed?

Reply:
Yes. I've rebased it on top of that change that just went in.